### PR TITLE
Remove the switch flag for dora write

### PR DESCRIPTION
**Pull request by huanghua78**
### What changes are proposed in this pull request?

Remove the switch flag for dora write. It is not needed any more.

### Why are the changes needed?

It is not needed any more. The write in dora is now enabled and tested. 

### Does this PR introduce any user facing changes?

No

### CODE REVIEW

**Review by yyongycy — APPROVED** [review]
lgtm

File: dora/core/common/src/main/java/alluxio/Constants.java [hunk 1]
```diff
@@ -234,11 +234,5 @@ public final class Constants {
   public static final String MEDIUM_HDD = "HDD";
   public static final String MEDIUM_SSD = "SSD";
 
-  /**
-   * Please use this switch enable/disable Dora write support in development.
-   * This will be removed when Dora write support is production ready.
-   */
-  public static final boolean ENABLE_DORA_WRITE = true;
```
Comment (jiacheliu3) [inline dora/core/common/src/main/java/alluxio/Constants.java:241]: fyi static final variables cannot be mocked so it's hard to test. A better way is using a method
```
public static boolean isWriteEnabled() {return ENABLE_DORA_WRITE;}
```
A static method can be mocked (although it's not the best way either). A property is probably the ideal solution.

Comment (huanghua78) [reply dora/core/common/src/main/java/alluxio/Constants.java:241]: This PR is to remove it.

This was introduced as a static switch when doing development.
No automation test is needed at that moment.
Manually changing the value is used in development cycle.

**Review by elega — APPROVED** [review]

**Comment by huanghua78** [comment]
alluxio-bot, merge this please

**Pull request merged**